src/pybrake/notifier.py

```diff
@@ -89,6 +89,7 @@ def __init__(
             "queue_stats": kwargs.get("queue_stats", True),
             "max_backlog_size": kwargs.get("max_backlog_size", 100),
             "backlog_enabled": kwargs.get("backlog_enabled", False),
+            "test_backlog_enabled": kwargs.get("test_backlog_enabled", False),
```

Comment:
It's not a good idea to add test options to public interfaces. I asked if it's possible, and looks like it's not. That's okay. Let's proceed without this option.